pkg/config/config.go

```diff
@@ -1095,6 +1095,14 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.path_merge.enabled", true)
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.traced_cgroups_count", 3)
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.traced_event_types", []string{"exec", "open", "dns", "bind"})
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.rate_exec", 100)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.burst_exec", 200)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.rate_open", 50)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.burst_open", 100)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.rate_dns", 10)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.burst_dns", 20)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.rate_bind", 10)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.burst_bind", 20)
```

Comment:
non-blocking nit: I'd probably suggest that this whole list of configs be sorted alphabetically. 


nit: Also, it may help users to swap the words of the last segment in the name and add separators for easy locating of variables. Eg:
```go
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.exec.rate", 100)
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.exec.burst", 200)
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.open.rate", 50)
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.open.burst", 100)
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.dns.rate", 10)
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.dns.burst", 20)
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.bind.rate", 10)
	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.bind.burst", 20)
```

Reply:
Thank you, I've just applied your suggestions